sklearn/base.py

```diff
@@ -349,8 +367,11 @@ def score(self, X, y, sample_weight=None):
         from .metrics import accuracy_score
         return accuracy_score(y, self.predict(X), sample_weight=sample_weight)
 
+    def _get_tags(self):
+        return _update_tags(self, super(ClassifierMixin, self),
```

Comment:
Does it matter that this always assumes that the class also inherits from `BaseEstimator` (or something defining `_get_tags`)?